File: aoj/volume100/ruby/A10014.rb
$<.each do |line|
  h, w = line.split.map(&:to_i)
  break if h == 0 && w == 0

  chess_board = 1.upto(h).map{|y|
    1.upto(w).map{|x|
      (x+y)%2 == 0 ? "#" : "."
    }.join + "\n"
  }.join + "\n"
  print chess_board
end

File: aoj/volume100/python3/A10014.py
import sys

for line in sys.stdin:
    h, w = map(int, line.rstrip().split())
    if h == 0 and w == 0:
        break

    rect = ''.join([
        ''.join([(x+y) % 2 == 0 and '#' or '.' for x in range(w)]) + '\n'
        for y in range(h)
        ])

    print(rect)
